README.md

```diff
@@ -1,35 +1,41 @@
-# odoh-client
+# odoh-client-go
```

Comment:
Somewhat opinionated: my feeling is that the README title should match the package name. I'll revert if you disagree.